packages/stack/test-runner/src/lib/index.js

```diff
@@ -158,6 +157,55 @@ class TestRunner {
     };
   }
 
+  setupGlobalVariables() {
```

Comment:
Duplicate of `setupGlobalVariables` method definition beginning on [L111](https://github.com/embark-framework/embark/blob/7f3ae16a232588b23f0e00614a7ae1c7fa849f96/packages/stack/test-runner/src/lib/index.js#L111), causes the linter to fail.